.github/workflows/continuous-integration.yml

```diff
@@ -16,17 +16,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-#        python-version: [3.6, 3.7, 3.8]
+#        python-version: [3.7, 3.8, 3.9, '3,10']
         include:
-        - os: ubuntu-18.04
-          python-version: 3.6
-
         - os: ubuntu-18.04
           python-version: 3.7
 
         - os: ubuntu-20.04
           python-version: 3.8
 
+        - os: ubuntu-20.04
+          python-version: 3.9
+
+        - os: ubuntu-20.04
+          python-version: '3.10'
+
```

Comment:
I have copied what SymPDE does for now. Considering ubuntu-18 usually doesn't come with python 3.7 and above it might be fine to keep it like this for now.

Reply:
Perfect, thanks